actions/transfer-case.vbs

```diff
@@ -274,13 +273,13 @@ ELSE 'this means out of county is TRUE '
 	LOOP UNTIL are_we_passworded_out = False					'loops until user passwords back in
 
     'Using move date to determine CRF change date.
-     cfr_date = dateadd("M", 1, client_move_date)
-     cfr_date = datepart("M", cfr_date) & "/01/" & datepart("YYYY", cfr_date)
-     cfr_date = dateadd("M", 2, cfr_date)
-     cfr_month = datepart("M", cfr_date)
-     IF len(cfr_month) <> 2 THEN cfr_month = "0" & cfr_month
-     cfr_year = datepart("YYYY", cfr_date)
-     cfr_year = right(cfr_year, 2)
+    cfr_date = dateadd("M", 1, client_move_date)
```

Comment:
This should be moved up to before the dialog and used to set the variables used in the dialog

Reply:
this cant be moved before the dialog cfr_date = dateadd("M", 1, client_move_date) but I can add date stuff before the dialog I will follow up with you tomorrow - since dates